docs/rules.md

```diff
@@ -363,6 +364,18 @@ These are considered invalid:
     end
 
 
+### NoInterpolationOnlyStrings
+
+A rule that disallows strings which are entirely the result of an
+interpolation.
+
+Good:
+    output = "Dogma.Rule.NoInterpolationOnlyStrings"
```

Comment:
The module doc currently breaks the rule. Make it a ~S""" sigil to prevent interpolation. :)

Also, I think the good and bad examples are switched.